packages/components/src/DateTimePickers/pickers/TimePicker/TimePicker.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+    padding: $padding-smaller 0;
+    background-color: $white;
+
+    button {
+        margin: 1px;
```

Comment:
Here I set margin as 1px to make focus outline visible when use keyborad up or down to move focus.